io.github.pixeldoted.cosmic-ext-color-picker.json

```diff
@@ -0,0 +1,49 @@
+{
+  "app-id": "io.github.pixeldoted.cosmic-ext-color-picker",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
```

Comment:
you should use the cosmic baseapp https://github.com/flathub/com.system76.Cosmic.BaseApp like the other cosmic apps.

It has themes and icons necessary to work on other desktops.